docs/tutorial/create_contact_page.md

```diff
@@ -132,6 +132,10 @@ To add contact information to your portfolio site, follow these steps:
 2. Add the necessary data.
 3. Publish your `Form Page`.
 
+```{note}
+If you want to receive your visitors' messages directly in your inbox, there are further configurations you must implement on your site.
```

Comment:
Well now that you have mentioned the possibility of getting the forms in my inbox, I want to know how. Is there a link we could send people to? Or does this information need to be added to this document? If so, what do you think about removing this note so we can merge everything else. Then re-add this note along with the configuration information as a separate PR.

Reply:
@cnk Thank you for your feedback. I've removed the note and will add detailed information or link(s) on how users can receive messages from their Form page directly in a separate pull request.